src/StreamFactoryInterface.php

```diff
@@ -9,15 +9,11 @@
     /**
      * Create a new stream from a resource or a string.
      *
-     * If the argument is a resource, it MUST be readable and SHOULD be seekable. It MAY be writable.
-     * If the argument is a string, a temporary resource will be created that is writable and seekable.
-     * 
-     * If the arugment is a string it will be interpided as the content of the stream. File names or 
-     * file paths are not supported. 
+     * String content will be used as the body of the stream. Files paths are not supported.
```

Comment:
@mindplay-dk are you opposed to having the user open the file with `fopen` and pass the resource in? It seems like there would be a lot of duplication in the implementation of these interfaces if every single implementation was doing:

```php
public function createFileStream($path, $mode = 'r')
{
    $resource = fopen($file, $mode);
    // ...
}
```